FrontEnd/src/actions/navigationActions.js

```diff
@@ -0,0 +1,8 @@
+export const SET_IS_DRAWER_OPEN = "SET_IS_DRAWER_OPEN";
```

Comment:
können wir das evtl nicht navigationAction/-Reducer  nennen sondern irgenwie anders? Denn wir haben schon teacherNavigation-xx und courseNavigation-xx... das werden sonst ganzschön viele ähnliche Namen. Vlt passt hier auch drawerAction/Reducer

Reply:
ich wollte halt einen Reducer haben, wo wir Sachen behandeln, die für Schüler und Lehreransicht wichtig sind. Wie zum Beispiel das  Drawer handling.
Da wird mittelfristig bestimmt noch was dazukommen, und dann will ich nicht noch einen weiteren Reducer hinzufügen (for obvious reasons)

Wie wärs mit generalNavigationReducer?

Reply:
Jaaa.. wird schon schiefgehen.. ich bin halt neulich schonmal drüber gestolpert, dass ich teachernavigationreducer und coursenavigationreducer verwechselt habe